src/lib/index/subscribe.js

```diff
@@ -35,31 +36,35 @@ const
 
 	emitter = new EventEmitter(),
 
-	subscribeAction = ({ topic, handler, channel }) => {
+	subscribeAction = ({ topic, handler, channel, config }) => {
 		// Ensure the topic exists
 		channel.assertQueue(topic);
-
+		// Set prefetch
+		if (config.prefetch && _.isInteger(config.prefetch)) {
```

Comment:
Is `config.prefetch = 0` a valid value? Because it's falsey...

Reply:
Technically yes, by the amqplib api spec, falsy values indicate 'unlimited' http://www.squaremobius.net/amqp.node/channel_api.html#channel_prefetch

Reply:
But this if won't allow channel.prefetch(0), even though the spec says it's a permitted value. Are you cool with that?